### DISCUSSION TITLE
Your GitHub Copilot access has been renewed - Student version

### DISCUSSION THREAD

**Discussion by blackrenard** [community/community#41423] [GitHub Education]
Hello everybody,

I'm a university student and about 2 months ago I activated the student version of GitHub. Reading online I saw that Copilot should be included within this plan. Last night for some reason, I got an error message saying I had to pay to use it, as I had finished the trial.
After a few minutes I received a mail that reads: "Your GitHub Copilot access has been renewed", saying that my version of Copilot will expire January 10, 2023 (the strange thing is that I didn't touch anything, it did everything by itself)

Why is there this deadline? Shouldn't it be active for the whole time I'm a student?

Thanks in advance for the help

**Comment by samus-aran** [comment]
Hi @blackrenard 👋 

Thank you for taking the time to report this issue.

Apologies for that temporary issue on our end. Please retry logging in to your IDE plugin, and Copilot should work as expected.

If you run into any further issues, please reply back in detail with any useful screenshots.
